spec.md

```diff
@@ -316,7 +316,7 @@ service Controller {
     returns (GetCapacityResponse) {}
 
   rpc ControllerGetCapabilities (ControllerGetCapabilitiesRequest)
-    returns (ControllerGetCapabilitiesResponse) {}  
+    returns (ControllerGetCapabilitiesResponse) {}
```

Comment:
Unrelated whitespace change?

Reply:
Yep but may be it is a good thing? This line had a trailing empty whitespace

Reply:
the "quality" of the change is fine. if this is to be included, please do so as a separate commit